wiki/en/Running-a-Server.md

```diff
@@ -193,37 +186,13 @@ Now you can run these with the `systemctl` command, for example:
 
 You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs.
 
-### Making a Server status page
-
-With the `-m` command line argument, Server statistics can be generated to be put on a web page.
-
-Here is an example php script using the Server status file to display the current Server status on a html page (assuming the following command line argument to be used: `-m /var/www/stat1.dat`):
-
-~~~
-<?php
-function loadserverstat ( $statfilename )
-{
-   $datei = fopen ( $statfilename, "r" );
-   while ( !feof ( $datei ) )
-   {
-          $buffer = fgets ( $datei, 4096 );
-          echo $buffer;
-   }
-   fclose($datei);
-}
-?>
-<?php loadserverstat ( "stat1.dat" ); ?>
-~~~
-
 
 # Servers on the desktop
 
-Jamulus can be configured to run in Server mode on the desktop. This gives you a graphical user interface to control most of the settings. 
+Jamulus can be configured to run in Server mode on the desktop. This gives you a graphical user interface to control most of the settings.
 
 * **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
-
 * **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
-
 * **Linux users** - Launch the "Jamulus Server" shortcut. Or you can open a terminal window (`CTRL+ALT+t` on Debian and related distros), type `jamulus -s` and hit return. _(To run a headless Server [read this guide](#installing-and-running-a-server))_
 
 
```

Comment:
Aha!!  What follows appears to start talking about configuration...

Reply:
yeah, this whole PR is broken. Ignore/

Reply:
Line 221 (new) `## Options` is a bit weird - L2 heading in the middle of a "list" of L3 headings.

Reply:
The following still need organising and merging with the command line options.

Reply:
> organising and merging with the command line options

I'd say that's a different PR to get the relevant CLUI options in among the GUI tab/screenshot stuff. Also implies we should do that for the client Software Manual too (but I hope we can keep the shared options in an include as today or we'll have FOUR places to keep them in sync) . 

@ann0see is just trying the shorted the overall thing here.

Reply:
> @ann0see is just trying the shorted the overall thing here.

Yes. I think another PR should be opened (@pljones could you please open an issue)

Reply:
Done https://github.com/jamulussoftware/jamuluswebsite/issues/729